bundles/org.openhab.binding.bondhome/src/main/resources/OH-INF/i18n/bondhome.properties

```
@@ -69,6 +69,8 @@ channel-type.bondhome.enableChannelType.label = Enable Up or Down Light
 channel-type.bondhome.enableChannelType.description = Enables or disables the up or down light of the ceiling fan. The light must also be on to turn on the up light.
 channel-type.bondhome.fanSpeedChannelType.label = Fan Speed
 channel-type.bondhome.fanSpeedChannelType.description = Sets fan speed
+channel-type.bondhome.rawFanSpeedChannelType.label = Raw Fan Speed
+channel-type.bondhome.rawFanSpeedChannelType.description = Set fan speed using the raw Bond values
```

Seems like regeneration was missed, in Channels.xml the text is:
```suggestion
channel-type.bondhome.rawFanSpeedChannelType.description = Sets fan speed using raw Bond values
```